node/broadcaster/types/config.go

```diff
@@ -37,6 +37,9 @@ type BroadcasterConfig struct {
 	// PendingTxToProcessedMsgs is the function to convert pending tx to processed messages.
 	PendingTxToProcessedMsgs PendingTxToProcessedMsgsFn
 
+	// HomePath is the path to the keyring.
+	HomePath string
+
 	// KeyringConfig is the keyring configuration.
 	KeyringConfig KeyringConfig
```

Comment:
maybe need to delete it?

Reply:
we need it to be used to get keyring base